common/persistence/cassandra/cassandraHistoryPersistence.go

```diff
@@ -350,7 +354,7 @@ func (h *cassandraHistoryV2Persistence) DeleteHistoryBranch(
 		BeginNodeID: common.Int64Ptr(beginNodeID),
 	})
 
-	rsp, err := h.GetHistoryTree(&p.GetHistoryTreeRequest{
+	rsp, err := h.GetHistoryTree(context.TODO(), &p.GetHistoryTreeRequest{
```

Comment:
why this one uses TODO, not the pass-in parameter?

Reply:
Yes it should use the pass-in parameter and I will change it to use the pass-in parameter in a later diff. For now, given that there're a large number of methods in persistence interface, I'd like to keep all methods implementing the persistence interface to use `_ context.Context` in the method definition, so that I can easily find all those methods later and won't miss any one.